tests/collector.py

```diff
@@ -9,7 +9,7 @@ def test_parse_indexes(self):
     oids = {(1, 2, 3, 4): 'eth0', (1, 3, 65, 32, 255): 'octet',
             (1, 4, 10, 181, 1, 30): '10.181.1.30',
```

Comment:
not sure what you meant by the missing length (4 is the length there), but I do see your point about making sure I am testing the right thing.